### PR TITLE
Add the site timezone info to the @site endpoint

### PR DESCRIPTION
**Pull request by folix-01**

### CODE REVIEW

**Comment by folix-01** [comment]
@jensens @mauritsvanrees  Hi, here I have the only test run that fails, it's the test for a python39. 
The problem is that the restapi response and the expected result have a different dictionary key-value order.
I don’t really understand why it happens, any idea? 
Screenshot 2024-02-16 at 17 27 48

**Comment by wesleybl** [comment]
@folix-01  did you run the tests locally and `plone.site_timezone` was `Europe/Berlin` in the resp file?

**Comment by stevepiercy** [comment]
@folix-01 the [CI's error message is above your screenshot with what happened and how to solve it](https://github.com/plone/plone.restapi/actions/runs/7933333589/job/21661857837#step:11:17).

**Comment by folix-01** [comment]
> @folix-01 did you run the tests locally and `plone.site_timezone` was `Europe/Berlin` in the resp file?

I didn't run the tests locally so as I can't make a buildout

**Comment by folix-01** [comment]
@jenkins-plone-org please run jobs

File: src/plone/restapi/services/site/get.py
Comment (erral) [inline src/plone/restapi/services/site/get.py:42]: I would say that we should get this setting from registry and not from the `default_timezone` function.

According to [its implementation](https://github.com/plone/plone.app.event/blob/master/plone/app/event/base.py#L449-L455) the function is designed to return the correct timezone even per-user, so if we call this endpoint with different users, we could get different results.

I would say that if we add the site_timezone setting, this should be the one set by the `plone.portal_timezone` setting.

I am OK on returning also the users own timezone, but I would return that in a different setting and do not call it "site_timezone" but "user_timezone".

**Review by erral — CHANGES_REQUESTED** [review]

File: src/plone/restapi/services/site/get.py
Comment (stevepiercy) [reply src/plone/restapi/services/site/get.py:42]: Renaming the key does not help. The implementation ambiguously returns either the user or site timezone. It would be better to name the key as `plone.timezone` because it is totally ambiguous from this implementation.

Thus I would go with @erral's suggestion. Get the value from the registry. However, I would return the user's timezone from the user endpoint, not the site endpoint.

File: src/plone/restapi/services/site/get.py
Comment (erral) [inline src/plone/restapi/services/site/get.py:45]: When adding the other settings (site_logo, robots.txt, etc) we kept the registry key name in the output JSON. I would do the same here and use `plone.portal_timezone` instead of `plone.timezone`

Comment (folix-01) [reply src/plone/restapi/services/site/get.py]: changed as required

File: src/plone/restapi/services/site/get.py
Comment (folix-01) [reply src/plone/restapi/services/site/get.py]: Ok, I agree, changed as required

**Review by stevepiercy — COMMENTED** [review]
LGTM, but I defer to @erral as he has greater familiarity with this code than I.

**Review by erral — APPROVED** [review]

**Comment by folix-01** [comment]
@erral @stevepiercy  How is the contribution workflow working? Do I have to do the merge personally or wait the release manager to do it?

**Comment by erral** [comment]
nope, @tisto or @davisagli will look at it and merge

**Comment by folix-01** [comment]
@jenkins-plone-org please run jobs

**Comment by folix-01** [comment]
@jenkins-plone-org please run jobs

**Comment by folix-01** [comment]
@jenkins-plone-org please run jobs

File: news/1749.feature
Comment (davisagli) [inline news/1749.feature:1]:
```suggestion
Add the site timezone to the @site endpoint return result. @folix-01
```

**Review by davisagli — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**
